Ryujinx.Ava/Ui/Backend/Vulkan/VulkanOptions.cs

```diff
@@ -0,0 +1,58 @@
+using System;
+using System.Collections.Generic;
+
+namespace Ryujinx.Ava.Ui.Vulkan
+{
+    public class VulkanOptions
+    {
+        /// <summary>
+        /// Sets the application name of the vulkan instance
+        /// </summary>
+        public string ApplicationName { get; set; }
+        
+        /// <summary>
+        /// Specifies the vulkan api version to use
```

Comment:
```suggestion
        /// Specifies the vulkan API version to use.
```
...etc, you get the idea. Won't comment on all of them.